extensions/datastores/accumulo/src/main/java/mil/nga/giat/geowave/datastore/accumulo/operations/config/AccumuloPasswordConverter.java

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
this class seems unnecessary (and is not currently used).  I think we can safely get rid of it.